test.sh

```diff
@@ -36,11 +36,11 @@ generate() {
 	protoc --go_gapic_out "$OUT" -I "$GOOGLEAPIS" $*
 }
 
-echo "Generating Cloud KMS v1"
-generate --go_gapic_opt 'go-gapic-package=cloud.google.com/go/kms/apiv1;kms,transport=grpc+rest' $GOOGLEAPIS/google/cloud/kms/v1/*.proto
+echo "Generating Cloud KMS v1 - gRPC"
+generate --go_gapic_opt 'go-gapic-package=cloud.google.com/go/kms/apiv1;kms,transport=grpc' $GOOGLEAPIS/google/cloud/kms/v1/*.proto
```

Comment:
why the switch to only one transport in this file?

Reply:
Because the problems i noticed arose when only rest was generated - with grpc+rest, the grpc code paths were including some imports that rest also needed, but when grpc was removed, the imports were not included. That is just one example. Thus, we want to make sure we do both in isolation. The following test in this file does both.